### ISSUE TITLE
Disaggregation Filter

### ISSUE THREAD

**Issue by WohnungNDS** [open-sdg/open-sdg#1207]
**Please complete the following information**
Data repository URL: [https://github.com/WohnungNDS/IM_Data/]
Data or metadata issue: Data
Input format: [e.g. CSV]

**Describe the bug / may be not a bug**

In [Indicator 4.1.3](https://wohnungnds.github.io/IM_Site/4-1-3/) I  have filled in some [testdata](http://prose.io/#WohnungNDS/IM_Data/edit/develop/data/indicator_4-1-3.csv)

With  indicator I want to show viditetd School and achieved grades depending on nationalty (French and German).

Everything works fine, If I chosse both nationalities an one school an on grade:

![image](https://user-images.githubusercontent.com/67795376/118946667-eaa5ea00-b956-11eb-8179-570f9e381515.png)

If I try to add a second School or a second grade one of the selected nationalities and school  dissapiers. in the example "BA, French A"

![image](https://user-images.githubusercontent.com/67795376/118947293-79b30200-b957-11eb-90d7-3ee4694a7e64.png)

I think it has to do with the  data strucutre. I have read the docs about [filter disaggregation](https://open-sdg.readthedocs.io/en/latest/open-sdg-features/#filter-by-disaggregation)

**Additional context**
This is only a test case. The real case whre i have observed this problem can be foun in [indicator 4-1-1](https://wohnungnds.github.io/IM_Site/4-1-1/) with a larger dadaset.

**Comment by brockfanning** [comment]
@WohnungNDS This seems similar to an issue that is (hopefully) fixed by #1205. Do you think you may be able to help us test that? If you want to test it out, you could temporarily change [this line](https://github.com/WohnungNDS/IM_Site/blob/develop/_config.yml#L28) to the following:

```
remote_theme: brockfanning/open-sdg@fix-for-get-combination-data
```

**Comment by WohnungNDS** [comment]
Great it seems to work!

![image](https://user-images.githubusercontent.com/67795376/118975625-7ed27a00-b974-11eb-9557-2f28559ec9fe.png)

will this fix be included in 1.4.0?

**Comment by WohnungNDS** [comment]
unfortunatly my "home" Logo dissapeared
Does this got to do with changing the remote theme to brockfanning/open-sdg@fix-for-get-combination-data

![image](https://user-images.githubusercontent.com/67795376/118989646-7da84980-b982-11eb-96a1-40845e5576b0.png)

**Comment by brockfanning** [comment]
@WohnungNDS Yes that is expected - to get the logo back you would also need to point to the 1.4.0-dev branch of the "jekyll-open-sdg-plugins" Ruby gem. You can do that by changing [this line](https://github.com/WohnungNDS/IM_Site/blob/develop/Gemfile#L7) to:

```
gem 'jekyll-open-sdg-plugins', git: 'https://github.com/open-sdg/jekyll-open-sdg-plugins.git', branch: '1.4.0-dev'
```

**Comment by WohnungNDS** [comment]
ok, the remote theme can be last as it is?

**Comment by WohnungNDS** [comment]
Great it works!
Thank you!
